include/solvers/adaptive_time_solver.h

```diff
@@ -22,7 +22,7 @@
 
 // Local includes
 #include "libmesh/system_norm.h"
-#include "libmesh/unsteady_solver.h"
```

Comment:
OK, sounds like a plan. I'll open an issue summarizing this once I finish up the other changes.